packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1092,20 +1092,83 @@ class Scaffold extends StatefulWidget {
   /// devices. Swipes in from either left-to-right ([TextDirection.ltr]) or
   /// right-to-left ([TextDirection.rtl])
   ///
-  /// In the uncommon case that you wish to open the drawer manually, use the
-  /// [ScaffoldState.openDrawer] function.
-  ///
   /// Typically a [Drawer].
+  ///
+  /// To open the drawer manually, use the [ScaffoldState.openDrawer] function.
+  ///
+  /// {@tool snippet --template=stateful_widget_material}
+  /// To disable the drawer edge swipe, set the [Scaffold.drawerEdgeWidth]
+  /// to 0. Then, use [ScaffoldState.openDrawer] to programmatically open the
```

Comment:
You can leave "programmatically" out here - since calling a method implies as much.